devices/stm32l0x0.yaml

```diff
@@ -0,0 +1,156 @@
+_svd: ../svd/stm32l0x0.svd
+
+_modify:
+  Flash:
+    name: FLASH
+  SYSCFG_COMP:
+    name: SYSCFG
+
+ADC:
+  CFGR1:
+    _modify:
+      AUTDLY:
+        name: WAIT
+  SMPR:
+    _modify:
+      SMPR:
+        name: SMP
+
+CRC:
+  POL:
+    _modify:
+      Polynomialcoefficients:
+        name: POL
+
+DBG:
+  _modify:
+    DBG_APB2_FZ:
+      name: APB2_FZ
+
+FLASH:
+  _modify:
+    OBR:
+      name: OPTR
+  _delete:
+    - WRPROT1
+    - WRPROT2
```

Comment:
I don't understand why these are being deleted here, since they seem to be correct and exist in RM0451.